src/main/java/org/reaktivity/nukleus/kafka/internal/stream/KafkaMergedFactory.java

```diff
@@ -1357,7 +1357,7 @@ private KafkaUnmergedProduceStream findProducePartitionLeader(
 
         private long initialId;
         private long replyId;
-        private MessageConsumer receiver;
+        private MessageConsumer receiver = (m, b, i, l) -> {};
```

Comment:
Recommend moving this no-op lambda value to a `static` constant `NO_RECEIVER` on the factory.